packages/react/src/hooks/useBreakpointValue.ts

```diff
@@ -0,0 +1,20 @@
+import { Breakpoint } from '../primitives/types/responsive';
+import { getValueAtCurrentBreakpoint } from '../primitives/shared/responsive/utils';
+import { useBreakpoint } from '../primitives/shared/responsive/useBreakpoint';
+import { useTheme } from './useTheme';
+
+export function useBreakpointValue<T>(
+  values: Record<string, T> | T[] | string,
```

Comment:
yes this argument type was just matching the type for the current useBreakpoint hook that we have but since we are only documenting the object and array syntax I think we can remove the string and just have the type be `Record<string, T> | T[]`